llvm/utils/vim/syntax/llvm.vim

```diff
@@ -270,3 +270,5 @@ if version >= 508 || !exists("did_c_syn_inits")
 endif
 
 let b:current_syntax = "llvm"
+
+setlocal commentstring=;\ %s
```

Comment:
Add this below `syn match llvmSpecialComment /`? Not many syntax files in `/usr/share/vim/vim90/syntax` set `commentstring`. When then do, they place `commentstring` near `syntax ...Comment`

Reply:
In https://github.com/LunarVim/LunarVim/issues/4394#issuecomment-1804542199, @chase mentions that it's more appropriate for this change to go in

llvm/utils/vim/ftplugin/llvm.vim

rather than

llvm/utils/vim/syntax/llvm.vim


Let me move this change there.